android/help/about/whatsnew.md

```diff
@@ -2,3 +2,6 @@
 title: What's New
 ---
 Here are some of the new features we have added to Keyman 17.0 for Android:
+
+* When suggestions aren't enabled, display a themed banner (#9696)
```

Comment:
I think we want to add:
* new gesture support
* removal of built-in browser (noting that it was a late 16.0 change also)
* performance improvements

Reply:
Ok. I referenced spec #5029 for the new gesture support. And  I split another section for "additional changes"